db.json

```diff
@@ -0,0 +1,215 @@
+      "editors": [
+        { "name": "Domenic Denicola", "email": "d@domenic.me" }
+      ]
+    },
+    {
+      "id": "infra",
+      "name": "Infra",
+      "scope": "The fundamental concepts upon which standards are built.",
+      "editors": [
+        { "name": "Anne van Kesteren", "email": "annevk@annevk.nl" }
+      ]
+    },
+    {
+      "id": "mimesniff",
+      "name": "MIME Sniffing",
+      "scope": "Algorithms used to determine the type of resources.",
+      "editors": [
+        { "name": "Gordon P. Hemsley", "email": "me@gphemsley.org" }
+      ]
+    },
+    {
+      "id": "notifications",
+      "name": "Notifications API",
+      "scope": "An API to display notifications to alert users outside the context of a web page.",
+      "editors": [
+        { "name": "Anne van Kesteren", "email": "annevk@annevk.nl" }
+      ]
+    },
+    {
+      "id": "quirks",
+      "name": "Quirks Mode",
+      "scope": "Quirks in CSS and Selectors that are necessary to support for web browsers for compatibility with the web.",
+      "editors": [
+        { "name": "Simon Pieters", "email": "zcorpan@gmail.com" }
+      ]
+    },
+    {
+      "id": "storage",
+      "name": "Storage",
+      "scope": "APIs for persistent storage and quota estimates, as well as the web platform storage architecture.",
+      "editors": [
+        { "name": "Anne van Kesteren", "email": "annevk@annevk.nl" }
+      ]
+    },
+    {
+      "id": "streams",
+      "name": "Streams",
+      "scope": "Web APIs for creating, composing, and consuming streams of data that map efficiently to low-level I/O primitives.",
+      "editors": [
+        { "name": "Domenic Denicola", "email": "d@domenic.me" }
+      ]
+    },
+    {
+      "id": "url",
+      "name": "URL",
+      "scope": "URLs, their infrastructure, and their API.",
+      "editors": [
+        { "name": "Anne van Kesteren", "email": "annevk@annevk.nl" }
+      ]
+    },
+    {
+      "id": "xhr",
+      "name": "XMLHttpRequest",
+      "scope": "A client-side networking API for transferring data between a web page and a server.",
+      "editors": [
+        { "name": "Anne van Kesteren", "email": "annevk@annevk.nl" }
+      ]
+    }
+  ],
+  "ideas": [
+    {
+      "title" : "CSS Books",
+      "ref-title": "BOOKS",
+      "href": "https://books.idea.whatwg.org/",
+      "description": "CSS Books defines CSS features for book publishing and high-quality printing of web pages.",
+      "authors": [
+        { "name": "Håkon Wium Lie" }
+      ]
+    },
+    {
+      "title": "CSS Figures",
+      "ref-title": "FIGURES",
+      "href": "https://figures.idea.whatwg.org/",
+      "description": "CSS Figures defines CSS features for floating elements around pages and columns.",
+      "authors": [
+        { "name": "Håkon Wium Lie" }
+      ]
+    }
+  ],
+  "references": {
```

Comment:
I'd call this something like `other_documents` and try to make the schema similar to the above, being an array with `ref_title`s inside, and `authors` being objects, not strings.